kubernetes/resource_kubernetes_persistent_volume_test.go

```diff
@@ -1162,6 +1205,43 @@ resource "kubernetes_persistent_volume" "test" {
 `, name)
 }
 
+func testAccKubernetesPersistentVolumeVolumeModeConfig(name, mode string) string {
+	return fmt.Sprintf(`
+resource "kubernetes_persistent_volume" "test" {
+  metadata {
+    annotations = {
+      TestAnnotationOne = "one"
+      TestAnnotationTwo = "two"
+    }
+
+    labels = {
+      TestLabelOne   = "one"
+      TestLabelTwo   = "two"
+      TestLabelThree = "three"
+    }
+
+    name = %[1]q
```

Comment:
I wasn't familiar with this notation, but it works. I tested it here: https://play.golang.org/p/691huK9ggSD

However, it probably makes sense to keep this consistent with our other tests by using `"%s"` when possible. The result is the same: https://play.golang.org/p/ETx99AiHMG6

```suggestion
    name = "%s"
```